src/Sentry.Tunnel/Sentry.Tunnel.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net5.0;netcoreapp3.0;netstandard2.0</TargetFrameworks>
```

Comment:
Never bothered with 2.1, only 3.0, not sure if there is a point to do 3.1 instead. I added net5.0 just to ensure we have a version that binds against ASP.NET Core 5, just in case. Not sure if we need it but gotta make sure it doesn't pull in weird, old dependencies.